dashboard/app/helpers/locale_helper.rb

```diff
@@ -34,6 +34,15 @@ def options_for_locale_select
     options
   end
 
+  def options_for_locale_code_select
+    options = []
+    Dashboard::Application::LOCALES.each do |locale, data|
+      next unless I18n.available_locales.include?(locale.to_sym) && data.is_a?(Hash)
```

Comment:
What are cases where `I18n.available_locales` doesn't include a given locale, or where the data isn't a hash?

Reply:
Available locales won't include [debug locales in production](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/application.rb#L94). If you looks at [locals.yml](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/locales.yml), half of the data isn't a hash. I copied most of this code from above but I'm now realizing that we can just iterate over the available_locales as we don't use anything in "data" for this function. I'll make that change.